deployment/cloud/aws/infrastructure/main.tf

```diff
@@ -8,7 +8,7 @@ terraform {
 
   backend "s3" {
     bucket = "${local.account_resource_prefix}-${var.account_id}-opentofu-state"
-    key = "infrastructure/${var.stage}/terraform.tfstate"
+    key    = "infrastructure/${local.neighborhood_lower}/${var.stage}/terraform.tfstate"
```

Comment:
changed this up in case there were multiple neighborhoods running in the same account. This lets their tfstate files live in separate locations :)